benchmark_runner/benchmark_operator/workload_flavors/perf_ci/uperf/uperf_data_template.yaml

```diff
@@ -30,13 +33,13 @@ shared_data:
   #metrics_profile: https://raw.githubusercontent.com/cloud-bulldozer/benchmark-operator/master/roles/kube-burner/files/metrics.yaml
 pod:
   es_kind: pod
-  client_limits_cpu: 4
-  client_limits_memory_GB: 16
-  server_limits_cpu: 4
-  server_limits_memory_GB: 16
+  client_limits_cpu: 8
+  server_limits_memory_GB: 8
```

Comment:
is that enough  8 GB RAM ?

Reply:
8GB is enough for uperf, I will confirm w/ a test run